packages/devtools_app/lib/src/screens/logging/logging_screen_v2/logging_screen_v2.dart

```diff
@@ -104,104 +85,71 @@ class _LoggingScreenState extends State<LoggingScreenBody>
   Widget build(BuildContext context) {
     return Column(
       children: [
-        _buildLoggingControls(),
-        const SizedBox(height: intermediateSpacing),
+        Row(
+          children: [
+            Expanded(
+              child: TextField(
+                decoration: const InputDecoration(
+                  border: OutlineInputBorder(),
+                  hintText: 'Filter',
+                ),
+                onSubmitted: (value) {},
+              ),
+            ),
+            Expanded(
+              child: TextField(
+                decoration: const InputDecoration(
+                  border: OutlineInputBorder(),
+                  hintText: 'Search',
+                ),
+                onSubmitted: (value) {},
+              ),
+            ),
+          ],
+        ),
         Expanded(
-          child: _buildLoggingBody(),
+          child: LogsTableV2(filteredLogs),
         ),
       ],
     );
   }
+}
 
-  // TODO(kenz): replace with helper widget
-  Widget _buildLoggingControls() {
-    final hasData = controller.filteredData.value.isNotEmpty;
-    return Row(
-      children: [
-        ClearButton(
-          onPressed: controller.clear,
-          gaScreen: gac.logging,
-          gaSelection: gac.clear,
-          minScreenWidthForTextBeforeScaling: loggingMinVerboseWidth,
-        ),
-        const Spacer(),
-        const SizedBox(width: denseSpacing),
-        // TODO(kenz): fix focus issue when state is refreshed
-        SearchField<LoggingControllerV2>(
-          searchFieldWidth: isScreenWiderThan(context, loggingMinVerboseWidth)
-              ? wideSearchFieldWidth
-              : defaultSearchFieldWidth,
-          searchController: controller,
-          searchFieldEnabled: hasData,
-        ),
-        const SizedBox(width: denseSpacing),
-        DevToolsFilterButton(
-          onPressed: _showFilterDialog,
-          isFilterActive: controller.isFilterActive,
-        ),
-        const SizedBox(width: denseSpacing),
-        CopyToClipboardControl(
-          dataProvider: () => controller.filteredData.value
-              .map((e) => '${e.timestamp} [${e.kind}] ${e.prettyPrinted()}')
-              .joinWithTrailing('\n'),
-          tooltip: 'Copy filtered logs',
-        ),
-        const SizedBox(width: denseSpacing),
-        SettingsOutlinedButton(
-          gaScreen: gac.logging,
-          gaSelection: gac.loggingSettings,
-          tooltip: 'Logging Settings',
-          onPressed: () {
-            unawaited(
-              showDialog(
-                context: context,
-                builder: (context) => const LoggingSettingsDialog(),
-              ),
-            );
-          },
-        ),
-      ],
+class LogsTableV2 extends StatelessWidget {
+  LogsTableV2(this.filteredLogs, {super.key});
+
+  final ScrollController _verticalController = ScrollController();
```

Comment:
remove type on LHS

Reply:
also I think this may need to be a stateless widget so the the scroll controller can be disposed